lib/libzfs/libzfs_import.c

```diff
@@ -88,6 +88,7 @@ typedef struct name_entry {
 	char			*ne_name;
 	uint64_t		ne_guid;
 	uint64_t		ne_order;
+	uint64_t		ne_labels;
```

Comment:
Perhaps `ne_labels` should instead be something like `ne_num_labels` to indicate its purpose.